core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -97,6 +95,11 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
   }
 
   ngAfterViewInit() {
+    const dialog = document.getElementById("mat-dialog-udf");
+    if (dialog !== null) {
+      dialog.style.width = "800px";
+      dialog.style.height = "600px";
```

Comment:
why are we setting width and height here instead of in CSS?

Reply:
The width and height in CSS will become the max-width and max-height of the modal. Seems like a limitation of the library.

Reply:
Understood. For those workarounds, it's better to leave a comment for future maintainers' information.

Reply:
Left comments.